Auth0/_ObjectiveAuthenticationAPI.swift

```diff
@@ -69,6 +76,18 @@ public class _ObjectiveAuthenticationAPI: NSObject {
                 }
             }
     }
+    
+    @objc(resumeAuthWithURL:options:)
+    public static func resume(_ url: URL, options: [A0URLOptionsKey: Any]) -> Bool {
+        return resumeAuth(url, options: options);
+    }
```

Comment:
This needs to be behind a `#if canImport(UIKit)` otherwise will fail to compile on macOS.

Reply:
Surrounded the function within the #if as requested